Client/Frontend/Browser/Tabs/SearchTermGroupsManager.swift

```diff
@@ -210,6 +209,10 @@ class SearchTermGroupsManager {
                 let firstTabTimestamp = firstTab.lastExecutedTime ?? firstTab.sessionData?.lastUsedTime ?? firstTab.firstCreatedTime ?? 0
                 let secondTabTimestamp = secondTab.lastExecutedTime ?? secondTab.sessionData?.lastUsedTime ?? secondTab.firstCreatedTime ?? 0
                 return firstTabTimestamp < secondTabTimestamp
+            } else if let firstSite = $0 as? Site, let secondSite = $1 as? Site {
```

Comment:
Nit:

```suggestion

            } else if let firstSite = $0 as? Site, let secondSite = $1 as? Site {
```

Vertical space is not at a premium and it makes it easier to differentiate between code paths. :)